src/components/widget/RawWidget.js

```diff
@@ -68,10 +68,11 @@ class RawWidget extends Component {
     handleFocus && handleFocus();
   };
 
-  willPatch = (value, valueTo) => {
+  willPatch = (value, valueTo, isForce) => {
```

Comment:
isn't it pointless to have that isForce parameter here just to return it?
why not better handle it in the caller?